packages/xrpl/test/client.ts

```diff
@@ -1,6 +1,5 @@
 import { assert } from 'chai'
 import _ from 'lodash'
-
```

Comment:
Why is this change necessary?

Reply:
Linter was unhappy - it's a result of running --fix